Modules/MUON/Common/src/TracksTask.cxx

```diff
@@ -145,49 +263,76 @@ bool TracksTask::assertInputs(o2::framework::ProcessingContext& ctx)
 
 void TracksTask::monitorData(o2::framework::ProcessingContext& ctx)
 {
-  ILOG(Info, Devel) << "Debug: MonitorData" << ENDM;
+  ILOG(Debug, Devel) << "Debug: MonitorData" << ENDM;
+
+  int firstTForbit = ctx.services().get<o2::framework::TimingInfo>().firstTForbit;
+  ILOG(Debug, Devel) << "Debug: firstTForbit=" << firstTForbit << ENDM;
 
   if (!assertInputs(ctx)) {
     return;
   }
 
-  ILOG(Info, Devel) << "Debug: Asserted inputs" << ENDM;
+  auto tracksMCH = ctx.inputs().get<gsl::span<o2::mch::TrackMCH>>("trackMCH");
+  auto clusters = ctx.inputs().get<gsl::span<o2::mch::Cluster>>("trackMCHTRACKCLUSTERS");
+
+  ILOG(Debug, Devel) << "Debug: Asserted inputs" << ENDM;
 
   mRecoCont.collectData(ctx, *mDataRequest.get());
 
-  ILOG(Info, Devel) << "Debug: Collected data" << ENDM;
+  ILOG(Debug, Devel) << "Debug: Collected data" << ENDM;
+
+  for (auto& p : mTrackPlotters) {
+    if (p.second) {
+      p.second->setFirstTForbit(firstTForbit);
+    }
+  }
+  for (auto& p : mTrackPlottersWithCuts) {
+    if (p.second) {
+      p.second->setFirstTForbit(firstTForbit);
+    }
+  }
 
   if (mSrc[GID::MCH] == 1) {
-    ILOG(Info, Devel) << "Debug: MCH requested" << ENDM;
-    if (mRecoCont.isTrackSourceLoaded(GID::MCH)) {
-      ILOG(Info, Devel) << "Debug: MCH source loaded " << mRecoCont.isTrackSourceLoaded(GID::MCH) << ENDM;
+    ILOG(Debug, Devel) << "Debug: MCH requested" << ENDM;
+    if (true || mRecoCont.isTrackSourceLoaded(GID::MCH)) {
```

Comment:
This `true` is correct? it looks like a leftover from debugging. I have the same comment for several other if statements below.

Reply:
Yes, inedeed. Removed.